spec/services/claims/fee_calculator/unit_price_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-RSpec.shared_examples 'a successful daily attendance fee calculation' do
+RSpec.shared_context 'a successful daily attendance fee calculation' do
```

Comment:
These are shared examples rather than shared contexts but the tests are using the custom `it_returns`, which is probably why there are Rubocop offences flagged as it appears that there are no tests in the block. The correct solution is probably either to add `it_returns` in the configuration as a recognised test or to use something that is more standard. I think that `it_returns` is just an alias for `it_behaves_like`.

Reply:
> These are shared examples rather than shared contexts but the tests are using the custom `it_returns`, which is probably why there are Rubocop offences flagged as it appears that there are no tests in the block. The correct solution is probably either to add `it_returns` in the configuration as a recognised test or to use something that is more standard. I think that `it_returns` is just an alias for `it_behaves_like`.

Doh. Good spot. I'll fix it.

Reply:
@jrmhaig I've fixed the issue you spotted by adding `it_returns` as an accepted rspec example in `.rubocop.yml`. This also had the rather nice side effect of removing 170 offences of another cop 🎉 

<img width="475" alt="image" src="https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/assets/28729201/6913dbaa-ec36-4297-b2ad-22a6b432bef3">

Can you re-review when you have time? No urgency. Thanks.